e2e/framework/provisioning/config_terraform.go

```diff
@@ -0,0 +1,85 @@
+package provisioning
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+)
+
+// ProvisionerConfigTerraform targets a Terraform cluster by reading the config
+// from a stdin, as output by 'terraform output provisioning'. Malformed inputs
```

Comment:
should `stdin` here be config file now?

Reply:
Yes, thanks. Leftover from my original failed attempts 😀

Reply:
Fixed in aa3b15d